docs/2.develop/contracts/environment/table-func.as.md

```diff
@@ -0,0 +1,14 @@
+<TableAsFunc>
+
+| Function Name          | SDK method                     | Description                                                            |
+| ---------------------- | ------------------------------ | ---------------------------------------------------------------------- |
+| SHA 256            | `context.sha256(value)`          | Hashes the random sequence of bytes using sha256.    |
+| Keccak 256        | `context.keccak256(value)`         | Hashes the random sequence of bytes using keccak256. |
```

Comment:
I think `random` is not the right word here, as the method is hashing a user-defined `value`. (which may or may not be random data)

Reply:
Agreed I've removed random from all three tables